.github/workflows/test.yml

```diff
@@ -285,14 +285,14 @@ jobs:
             runner: windows-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: ./.github/actions/build-debug-turborepo
+      - uses: ./.github/actions/setup-turborepo-environment
         with:
-          github-token: "${{ secrets.GITHUB_TOKEN }}"
           target: ${{ matrix.os.name }}
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
 
       - name: E2E Tests
         # Turbo has already been built in previous step, no need to rebuild
-        run: pnpm -- turbo-prebuilt run e2e-prebuilt --filter=cli
+        run: pnpm -- turbo run e2e-prebuilt --filter=cli
```

Comment:
So this will build turbo individually right?

Reply:
We basically build a first version of turbo to then run our individual tasks, some of which involve building turbo again. We do this on every single workflow so that's a lot of redundant turbo building. It's like building rustc from source for every workflow run.